sources/packages/api/src-sql/Restrictions/Add-col-action-type-and-notification-type.sql

```diff
@@ -0,0 +1,15 @@
+-- Add action_type to restrictions table.
+ALTER TABLE
+  sims.restrictions
+ADD
+  COLUMN IF NOT EXISTS action_type sims.restriction_action_type [] NOT NULL DEFAULT ARRAY ['No effect'] :: sims.restriction_action_type [];
+
+COMMENT ON COLUMN sims.restrictions.action_type IS 'List of enum restriction_action_type, i.e this column contains the list of action types for a restrictions.';
```

Comment:
This is inferred by the column type "List of enum restriction_action_type", can the comment be something like:
```
Actions associated with the restriction, for instance, when a restriction must prevent the student from applying to a Student Application.
```